src/EditorFeatures/TestUtilities2/Utilities/GoToHelpers/MockNavigableItemsPresenter.vb

```diff
@@ -1,27 +1,35 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+Imports System.Composition
 Imports System.Threading
 Imports Microsoft.CodeAnalysis.Editor.FindUsages
 Imports Microsoft.CodeAnalysis.Editor.Host
 Imports Microsoft.CodeAnalysis.FindUsages
+Imports Microsoft.CodeAnalysis.Host.Mef
 
 Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Utilities.GoToHelpers
+    <Export(GetType(IStreamingFindUsagesPresenter))>
+    <[Shared]>
+    <PartNotDiscoverable>
     Friend Class MockStreamingFindUsagesPresenter
```

Comment:
File name desn't match the type name. I'd give you a pass but since you're introducing a second MockStreamingFindReferencesPresenter I'm a bit less inclined to be nice.

Reply:
This file is no longer touched as part of this pull request.